### ISSUE TITLE
1.1.2 issue with height double.infinity

### ISSUE THREAD

**Issue by calvinmuller** [renefloor/flutter_cached_network_image#264]
This used to work with 1.1.1

```
            child: CachedNetworkImage(
              fadeInDuration: Duration(),
              imageUrl: organisation.images.first.image.getSmartThumb(),
              fit: BoxFit.cover,
              height: height,
              width: double.infinity,
            ),
```
However in 1.1.2 it's producing `Unsupported operation: Infinity or NaN toInt`

**Comment by renefloor** [comment]
You are right. I published a fix as 1.1.2+1

**Comment by yrom** [comment]
Hi, @renefloor. It was wrong to use the '==' operator to check 'NaN'.
